### DISCUSSION TITLE
최준철 - SNS - Youtube 참여 지표 분석을 통한 나라별 청중 선호도 차이

### DISCUSSION THREAD

**Discussion by jjeori** [dinnercoding/wanted-pre-onboarding-challenge-data#104] [2023년 11월 프리온보딩 데이터 챌린지]
### 도메인 선정 이유
* 좋아요 수, 댓글 수, 조회 수 등 어떤 참여 지표가 인기 동영상에 영향을 미치는지 궁금하여 해당 도메인을 선택하였다.
* 더 나아가 각 나라마다 참여 지표의 차이가 있는지 확인하고자 한다.

### 문제 정의
* 사람마다 영상에 호감을 표현하는 방법은 다르다.
* 각 나라마다 참여 지표의 차이가 있다면, 그 원인은 무엇인지 파악하고자 한다.
* 어떤 참여 지표가 인기 동영상에 영향을 미치는지 파악하다면, 영상 시청자들에게 특정 지표에 참여를 유도하는 전략을 펼칠 수 있을 것이다.

### 데이터 업로드
* [일일 트렌드 YouTube 동영상(113개국)](https://www.kaggle.com/datasets/asaniczka/trending-youtube-videos-113-countries)
* 출처: Kaggle

**Comment by ghost** [comment]
준철님! 문제 정의 잘 해주셨습니다~ 국가가 113개이다 보니 범위가 너무 넓어 시각화할 때 약간의 어려움이 있을 것 같아요.
한국과 다른 나라 몇개국 정도로 범위를 좁혀보셔도 좋을 것 같다는 생각이 듭니다.
더 좋은 아이디어가 있다면 언제든지 환영입니다~ ㅎㅎ
